apps/backend/src/db/migrations/20240823115158_create_onboarding_questions_table.ts

```diff
@@ -0,0 +1,31 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = "onboarding_questions";
+
+const ColumnName = {
+	CREATED_AT: "created_at",
+	ID: "id",
+	QUESTION: "question",
```

Comment:
```suggestion
	LABEL: "label",
```
Let's have the same as with quiz questions